frontend/src/fixedComponents/Slider.js

```diff
@@ -1,15 +1,61 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import logo1 from '../images/1.jpg';
+import logo2 from '../images/2.jpg';
+import logo3 from '../images/3.jpg';
 
-export const Slider = (props) => {
+ const Slider = () => {
   return(
-  <div>
-      
-  </div>);
-};
-
-const mapStateToProps = (state) => ({});
+<div>
+    <div id="carouselExampleIndicators" className="carousel slide" data-ride="carousel" style={{height: "500px"}}>
+        <ol className="carousel-indicators">
+          <li data-target="#carouselExampleIndicators" data-slide-to="0" className="active"></li>
+          <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+          <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+          <li data-target="#carouselExampleIndicators" data-slide-to="3"></li>
+        </ol>
+        <div className="carousel-inner">
```

Comment:
Instead of setting up a separate structure for each image, you can create a loop.